Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1824,6 +1850,17 @@ extension BrowserViewController: ReaderModeDelegate, UIPopoverPresentationContro
     }
 }
 
+// MARK: - UIPopoverPresentationControllerDelegate
```

Comment:
Ya the BrowserViewController is quite the unruly beast. Thanks for breaking this out into a separate extension. We're looking into ways to simplify all of this code and having things broken out into smaller extensions helps alot!